scripts/overrideMasterportalapi.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+/*
+ * NOTE
+ *
+ * In current masterportalapi versions, this behaviour can be configured.
+ * Since it currently can't, we'll deactive the wildcard on street names. It
+ * may lead to notoriously bad results on an address search without further
+ * filtering, e.g. "Roonstraße" will first result in the "Liliencronstraße",
+ * which is about as confusing as it gets.
+ *
+ * On an update to the masterportalapi, we may make use of this by first
+ * running on the method without wildcard and, if no results came in, with
+ * wildcard.
```

Comment:
This part should be added to the `arcana.md` instead of describing it here.

Reply:
This particular line is more of a development ticket that's currently not written out. One would automatically find it via the `arcana.md` by searching for the package as described. However, an additional note does not hurt: https://github.com/Dataport/polar/pull/90/commits/76129bb0e003ab407328d0d49ba9da6d14797194